### ISSUE TITLE
A small mistake in the document of CorrectAndSmooth

### ISSUE THREAD

**Issue by swy9834** [rusty1s/pytorch_geometric#2862]
I think there is  a small mistake in the math formula below, where **Z** is the soft predictions and **Y** is the ground-truth training label information.
![2021-07-15 11-39-04 的屏幕截图](https://user-images.githubusercontent.com/30220798/125725493-b89f86fa-a9cc-497f-aff3-3278b4b32ae6.png)
Here is the related code, using **y-z** instead of **z-y**
```
error = torch.zeros_like(y_soft)  
error[mask] = y_true - y_soft[mask] 
```
I believe the math formula should be
![2021-07-15 11-59-21 的屏幕截图](https://user-images.githubusercontent.com/30220798/125727174-973e667e-ebae-40fa-b7e4-2815f243bf3e.png)
Thanks for your hard working and this awesome framwork.

**Comment by rusty1s** [comment]
Thanks! I fixed it in the documentation.
